utilities/rocAL/rocAL_basic_test/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../amd_openvx/cmake)
 find_package(OpenCV QUIET)
 find_package(AMDRPP QUIET)
 
-include_directories(${ROCM_PATH}/libexec/mivisionx/rali/python/)
```

Comment:
scripts need to install at -- ${CMAKE_INSTALL_LIBEXECDIR}/mivisionx/rocAL

Reply:
Done